packages/maskbook/src/plugins/PluginUI.ts

```diff
@@ -16,8 +16,6 @@ import { DHedgePluginDefine } from './dHEDGE/define'
 
 sideEffect.then(() => {
     if (Flags.ito_enabled) plugins.add(ITO_PluginDefine)
-    if (Flags.gitcoin_enabled) plugins.add(GitcoinPluginDefine)
-    if (Flags.poll_enabled) plugins.add(PollsPluginDefine)
```

Comment:
@Jack-Works is need delete unused flags?

Reply:
yes